test/lib.test.js

```diff
@@ -96,6 +96,25 @@ describe('can be used as a library', () => {
       done();
     });
   });
+  it('should be able to crop and then upload an image using the gravity option', (done) => {
+    const stream = fs.createReadStream(testImage);
+    const options = {
+      imagemagick: useImageMagick,
+      bucket: process.env.AWS_BUCKET,
+      profile: process.env.AWS_PROFILE,
+      quality: 80,
+      position: [10, 10],
+      size: [120, 120],
+      gravity: 'Center'
+    };
+    s3put(stream, options, (err, response) => {
+      if (err) {
+        console.log(err);
+      }
+      chai.expect(response.key).to.include(testImageBase);
```

Comment:
Not really.  I started to write code to retrieve the image back off s3 here, to compare it to a local expected copy, but decided to just leave it.  Let me know if you want to just remove this test, or have me put in the image verifier (which we can re-use to confirm the other test cases as well).